spec/std/json/serialization_spec.cr

```diff
@@ -169,7 +169,7 @@ describe "JSON serialization" do
     end
 
     it "deserializes Time" do
-      Time.from_json(%("2016-11-16T09:55:48-0300")).to_utc.should eq(Time.utc(2016, 11, 16, 12, 55, 48))
+      Time.from_json(%("2016-11-16T09:55:48-03:00")).to_utc.should eq(Time.utc(2016, 11, 16, 12, 55, 48))
```

Comment:
We shouldn't break this

Reply:
According to https://github.com/crystal-lang/crystal/pull/5123#discussion_r193401753
I agree. The JSON format should be ISO8601 by default.

Reply:
Output should be the RFC 3339 flavour of ISO 8601, the parser should accept more variants. That's probably the best solution.